### DISCUSSION TITLE
Compile time error.

### DISCUSSION THREAD

**Discussion by V-Mohammad-taqi** [shadowsocks/shadowsocks-android#2875] [General]
java.lang.IllegalAccessError: class org.jetbrains.kotlin.kapt3.base.KaptContext (in unnamed module @0x1d312b6c) 
cannot access class com.sun.tools.javac.util.Context (in module jdk.compiler)
 because module jdk.compiler does not export com.sun.tools.javac.util to unnamed module @0x1d312b6c
	at org.jetbrains.kotlin.kapt3.base.KaptContext.<init>(KaptContext.kt:28)

**Comment by itsthekeming** [comment]
Did you ever identify a solution to this issue? It's popped in a Spring project of mine and I can't for the life of me figure it out.
